API/Resources/ProjectResultResource.cs

```diff
@@ -41,6 +42,11 @@ public class ProjectResultResource
         /// </summary>
         public string ShortDescription { get; set; }
 
+        /// <summary>
+        ///     This gets or sets the collaborators
+        /// </summary>
+        public ICollection<CollaboratorResourceResult> Collaborators { get; set; }
```

Comment:
Is there a reason why you would use ICollection over IEnumerable? I could be overlooking a thing of course. 

However, you could say the following:
- Use IEnumerable: for a list of objects that only needs to be iterated through
- Use ICollection: for a list of objects that needs to be iterated through and modified

And I don't think we need to modify them right?